syntax = "proto3";

package user;

service UserService {
  rpc GetUserStatus (UserRequest) returns (UserResponse);
}

message UserRequest {
  string user_id = 1;
}

message UserResponse {
  string status = 1;
}
